pokemon-persistence/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/dao/PokemonDAO.java

```diff
@@ -52,4 +52,11 @@
      */
     List<Pokemon> getAllPokemon();
 
+    /**
+     * Returns a list of all pokemons of a trainer.
+     * 
+     * @param trainer The trainer to find the pokemons of.
+     * @return All pokemons of the trainer.
+     */
+    List<Pokemon> getAllPokemonsOfTrainer(Trainer trainer);
```

Comment:
Plural of Pokemon issue again - for sake of consistency in naming, we should drop All from the method name.